components/GothamistMainHeader.vue

```diff
@@ -1,134 +1,166 @@
 <script setup lang="ts">
-import GothamistSidebarContents from "./GothamistSidebarContents.vue";
-import Navigation from "~~/composables/types/Navigation";
+import GothamistSidebarContents from './GothamistSidebarContents.vue'
+import Navigation from '~~/composables/types/Navigation'
 import VFlexibleLink from '@nypublicradio/nypr-design-system-vue3/v2/src/components/VFlexibleLink.vue'
 
 defineProps<{
-    showLogo: boolean,
-    navigation: Navigation
-    donateUrl: string
+  showLogo: boolean
+  navigation: Navigation
+  donateUrl: string
 }>()
 
-const sideBarVisible = ref(false);
+const sideBarVisible = ref(false)
+
+const menuListClick = (menuTitle) => {
+  //emitted mobile menu click event
+  // sendEvent('click_tracking', {
```

Comment:
added click tracking place holder

Reply:
You can use the analytics plugin for this
load it with `const { $analytics } = useNuxtApp()`
and then call `$analytics.sendEvent` instead of just `sendEvent`